lib/backend.js

```diff
@@ -9,16 +9,52 @@ const pickBy = require('lodash/pickBy')
 const omit = require('lodash/omit')
 const querystring = require('querystring')
 const clc = require('cli-color')
-
 require('es7-object-polyfill')
 
+process.env.AWS_SDK_LOAD_CONFIG=1
```

Comment:
See my second point from previous comment. I'd like to avoid modifying global environment and there is an API to read credentials from file without setting this variable.

Reply:
Let me have a look again.

Reply:
I stumbled upon a doc which explains how the AWS_SDK_LOAD_CONFIG var works.
https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html

It looks to me that we can set this var without any concern as it is the last in the order of precedence for loading the region.

So even if we set the AWS_SDK_LOAD_CONFIG to true this will not effect an existing project since they will most likely override AWS_REGION using an env variable or programmatically.

What are your thoughts on this ?

Reply:
I checked the SDK and it seems to load automatically the credentials file but not the config file (region is in ~/.aws/config and not in ~/.aws/credentials). I even went on looking at the sources. 

You have a valid point about the lib use case, but I think if we use it as a lib we would already set the region programmatically or via env var. Or worse if no region is set we end up with default region us-east-1. So nothing should be broken but we at least fix the problem for people who wanted to use it as a standalone tool (most of the use cases ?)

Reply:
OK, then we have no option but to keep it.

But please move this env assignment and require('aws-sdk') back to where it was initially so that we don't unnecessarily trigger loading when using this programatically.